ghost/verification-trigger/lib/verification-trigger.js

```diff
@@ -19,7 +19,7 @@ class VerificationTrigger {
      * @param {number} deps.importTriggerThreshold Threshold for triggering Import sourced verifications
      * @param {() => boolean} deps.isVerified Check Ghost config to see if we are already verified
      * @param {() => boolean} deps.isVerificationRequired Check Ghost settings to see whether verification has been requested
-     * @param {(content: {subject: string, message: string, amountTriggered: number}) => void} deps.sendVerificationEmail Sends an email to the escalation address to confirm that customer needs to be verified
+     * @param {(content: {subject: string, message: string, amountTriggered: number}) => Promise<void>} deps.sendVerificationEmail Sends an email to the escalation address to confirm that customer needs to be verified
```

Comment:
Should be a promise so we can properly await sending an email. If we don't want to await, that decision should be made inside the verification trigger class and not elsewhere. But for testing awaiting is easier and doesn't require a hacky timeout.